Production/crab/crab_tools.py

```diff
@@ -57,7 +57,13 @@ def __init__(self, file_name, job_names = '', lumi_mask = '', jobNameSuffix = ''
         if len(lines) <= 2:
             raise RuntimeError("file '{}' is empty".format(file_name))
         header_items = filter(lambda s: len(s) != 0, re.split(" |\n", lines[0]))
-        self.pyCfgParams = filter(lambda s: len(s) != 0, re.split(" |\t", lines[1]))
+        if header_items[0].startswith("lumiMask"):
+            self.pyCfgParams = filter(lambda s: len(s) != 0, re.split(" |\t", lines[1]))
+            lumi = filter(lambda s: len(s) != 0, re.split("=", header_items[0]))
+            self.lumiMask = lumi[1]
+        else:
+            self.lumiMask =  ''
+            self.pyCfgParams = filter(lambda s: len(s) != 0, re.split(" |\t", lines[0]))
```

Comment:
don't duplicate pyCfgParams initialization, just change cfgParamsIndex inside the if/else.